libparsec/crates/protocol/tests/authenticated_cmds/v3/user_get.rs

```diff
@@ -1,19 +1,143 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+// `allow-unwrap-in-test` don't behave as expected, see:
+// https://github.com/rust-lang/rust-clippy/issues/11119
+#![allow(clippy::unwrap_used)]
+
+use libparsec_tests_lite::prelude::*;
+
 use super::authenticated_cmds;
 
 // Request
 
 pub fn req() {
-    // TODO #4545: Implement test
+    // Generated from Python implementation (Parsec v2.6.0+dev)
+    // Content:
+    //   cmd: "user_get"
+    //   user_id: "109b68ba5cdf428ea0017fc6bcc04d4a"
+    let raw = hex!(
+        "82a3636d64a8757365725f676574a7757365725f6964d92031303962363862613563646634"
+        "32386561303031376663366263633034643461"
+    );
+
+    let req = authenticated_cmds::user_get::Req {
+        user_id: "109b68ba5cdf428ea0017fc6bcc04d4a".parse().unwrap(),
+    };
+
+    let expected = authenticated_cmds::AnyCmdReq::UserGet(req);
+
+    let data = authenticated_cmds::AnyCmdReq::load(&raw).unwrap();
+
+    p_assert_eq!(data, expected);
+
+    // Also test serialization round trip
+    let authenticated_cmds::AnyCmdReq::UserGet(req2) = data else {
+        unreachable!()
+    };
+
+    let raw2 = req2.dump().unwrap();
+
+    let data2 = authenticated_cmds::AnyCmdReq::load(&raw2).unwrap();
+
+    p_assert_eq!(data2, expected);
 }
 
 // Responses
 
 pub fn rep_ok() {
-    // TODO #4545: Implement test
+    let raws = [
```

Comment:
Merge `raws` and `expected` to form a tuple list ?

Reply:
why not

Reply:
Done